kemitix/kemitix-quality-maven-plugin
# Release 0.2.0

* Add `org.pitest:pitest-maven:mutationCoverage`
* Add `org.pitest:highwheel-maven:analyse`
* Add `jacoco-maven-plugin:{prepare-agent,check}`
* Allow disabling individual plugin goals
* Add `huntbugs-maven-plugin:huntbugs`
* Document properties in `README.md`
* Require code coverage of 100% (excluding the `mojo-executor` module which wraps a static method)
* Add `maven-enforcer-plugin:{display-info,enforcer}`
* Add support for multiple maven goals for each `PluginGoal`
